dependenciesWhitelist.txt

```diff
@@ -40,6 +40,7 @@ egg
 electron
 eventemitter2
 eventemitter3
+expect
```

Comment:
a bunch of other jest packages also has types now, but I guess this list should only contain the packages we know we need and not everything that's theoretically possible?

Reply:
Yes, basically.